pages/options/options.html

```
@@ -986,7 +986,7 @@ <h3>存儲使用量</h3>
             ></output>
 
             <!-- 執行清理按鈕（僅在有可清理項目時顯示） -->
-            <button id="execute-cleanup-button" class="btn-danger" style="display: none">
+            <button id="execute-cleanup-button" class="btn-danger hidden">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)